src/client/src/types/referenceDataService.ts

```diff
@@ -0,0 +1,7 @@
+import { Observable } from 'rxjs/Rx'
```

Comment:
So, we've got 2 files with the same name now in the project - referenceDataService.ts ?

Reply:
I would suggest to keep it like this now and fix it later. Are you happy with that? I'm doing another story that will clean up many things.

Reply:
sounds good

Reply:
thanks